### PR TITLE
DOC: Removing tutorials from sphinx documentation

### LINKED ISSUE #19191: DOC: Discussion - How to organize tutorials
## Documentation

Now that https://numpy.org/numpy-tutorials/ is deployed, we need to decide how to organize our internal documentation. 

Currently, [this page](https://numpy.org/devdocs/user/tutorials_index.html) is also called NumPy Tutorials. The two tutorials listed there are also available as Jupyter Notebooks from the [separate numpy-tutorials repo](https://github.com/numpy/numpy-tutorials).

In our docs team meetings, we have discussed two possibilities:

1.  Keeping two sets of tutorials. This would allow us to separate the contents into in-depth guides about NumPy features (in the main sphinx docs) and general tutorials with applications of NumPy in other domains (in the numpy-tutorials repo). In this case, the tutorials section in the main docs would have to be renamed; one suggestion is "NumPy Guides" or similar.
2. Only keeping the numpy-tutorials repo, making the sphinx tutorials page point to the rendered numpy-tutorials site and adding any new content there.

It would be nice to have further input from the community so we can reach a solution. If you have other ideas, please feel free to let us know!

cc @rossbar @rgommers @cooperrc @8bitmp3

### PR DESCRIPTION
**Pull request by melissawm**
Also added links to the same content now hosted at https://numpy.org/numpy-tutorials

(Following the discussion on #19191 and the documentation team meetings).

Closes #19191

### CODE REVIEW

File: doc/source/user/tutorials_index.rst [hunk 1]
```diff
@@ -5,12 +5,8 @@ NumPy Tutorials
...
-.. toctree::
-   :maxdepth: 1
-
-   tutorial-svd
-   tutorial-ma
+features, as well as a few applications. You can find them 
+`in the NumPy Tutorials page <https://numpy.org/numpy-tutorials>`_.
 
+For detailed reference documentation of the functions and
+classes contained in the package, see the :ref:`API reference <reference>`.
```
Comment (mattip) [inline doc/source/user/tutorials_index.rst:12]: Hmm. This doesn't really flow well. Maybe something like

Tutorials about NumPy are hosted at https://numpy.org/numpy-tutorials. For how-tos, reference documents, and explanations see the ref:`API reference <reference>` documentation.

Comment (melissawm) [reply doc/source/user/tutorials_index.rst]: Fixed, thanks!

**Comment by mattip** [comment]
Thanks. Now that I see the rendered page https://20803-908607-gh.circle-artifacts.com/0/doc/build/html/user/tutorials_index.html I am not sure it deserves a page to itself.  Maybe it should be removed and the index page link should point to the numpy.org page with some visual indication that it is "off site"?

**Comment by melissawm** [comment]
Yeah, I agree. Let me change this to just be a link at the sidebar for now, and I guess it will have its own card when the new landing page is done.

**Comment by melissawm** [comment]
[This is how the preview looks like](https://20908-908607-gh.circle-artifacts.com/0/doc/build/html/user/index.html) - I hadn't noticed before but the theme already includes the "external link" symbol 😄

**Comment by mattip** [comment]
Thanks @melissawm

**Pull request merged**